docs/src/examples.rst

```diff
@@ -14,205 +14,175 @@
 |cdk| Examples
 ##############
 
-This topic contains some usage examples to help you get started understanding
-the |cdk|.
+This topic contains some examples to help you get started using some of the advanced constructs
+offered by the |cdk|.
 
-.. We'll include this if we ever implement DeploymentPipeline
-   _multiple_stacks_example:
+.. _creating_ecs_l2_example:
 
-   Creating an App with Multiple Stacks
-   ====================================
+Creating an |ECS| Construct
```

Comment:
I wouldn't call it "creating an ECS *construct*", that implies you're building a reusable thing.

Maybe "Writing an |ECS| CDK app" ?